spec/models/retirement_manager_spec.rb

```diff
@@ -1,5 +1,9 @@
 describe RetirementManager do
   describe "#check" do
+    before do
+      User.super_admin || FactoryGirl.create(:user_with_group, :userid => "admin")
```

Comment:
I don't think simply naming a user with `admin` will give us a super admin anymore.
Could you see if this works?

```suggestion
      User.super_admin || FactoryGirl.create(:user, :userid => "admin", :features => :everything`)
```

of course, I'd prefer we didn't use "admin" and used an explicit user instead but... this may be a simpler fix.

Reply:
😆 Sorry, this check is at the highest level before a user is required anyway...